### PR TITLE
Docs: Experiment CLI Common Params

### PR DESCRIPTION
**Pull request by melo-gonzo**
This PR adds some notes on commonly updated params in the config files for the experiment CLI.

### CODE REVIEW

File: docs/source/experiment-interface.rst [hunk 1]
```diff
@@ -254,3 +254,82 @@ Only arguments which contain dict: \[str, int, float\] mapping all the way
...
+
+
+Experiment CLI and Downstream Tasks
+-----------------------------------
+
+The experiment cli provides some helpful utilities to enable downstream task development by removing boilerplate which would otherwise become time consuming and hard to track. Specifically, loading models, datasets, and tasks from yaml files makes it simple to pop in and out different transforms, callbacks, and datasets by simply updating yaml files instead of parsing out relevant sections of code. For example, to quickly load in a model and associated transforms, the following code block may be used:
+
+.. code-block:: python
+  :caption: Load in transforms from a yaml file
+    import yaml
```
Comment (laserkelvin) [inline docs/source/experiment-interface.rst:327]: Does this render correctly? IIRC you need a blank line between the caption and the actual code

File: docs/source/experiment-interface.rst [hunk 2]
```diff
@@ -254,3 +254,82 @@ Only arguments which contain dict: \[str, int, float\] mapping all the way
 through to the target value may be updated. Parameters which map to lists at any
 point are not updatable through ``cli_args``, for example callbacks, loggers, and
 transforms.
+
+Commonly Updated Parameters
+---------------------------
+
+Parameters that are frequently updated are highlighted below and sectioned off by thier respective configs. To see a full set of parameters, it is recommended to navigate to the modules specific documentation.
```
Comment (laserkelvin) [inline docs/source/experiment-interface.rst:261]: Typo on "thier"

File: docs/source/experiment-interface.rst [hunk 3]
```diff
@@ -254,3 +254,82 @@ Only arguments which contain dict: \[str, int, float\] mapping all the way
...
+
+Common Trainer Config Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Anything that would go in ``pl.Trainer()`` can be found here, including hardware setup, number of epochs to train for, callbacks, and loggers.
+
+.. code-block:: yaml
+
+  min_epochs
```
Comment (laserkelvin) [inline docs/source/experiment-interface.rst:270]: This previews oddly, so I'm worried it'll also look odd in the built docs. Can you add nominal values to each one?

```yaml
min_epochs: 0
max_epochs: 100
accelerator: gpu          # can be cpu, gpu, xpu
strategy: null            # can be ddp, ddp_with_unused_parameters, etc.
```

File: docs/source/experiment-interface.rst [hunk 4]
```diff
@@ -254,3 +254,82 @@ Only arguments which contain dict: \[str, int, float\] mapping all the way
...
+.. code-block:: yaml
+
+  batch_size
+  num_workers
+  train_path
+  val_split
+  test_split
+  target_keys
+  task_args
+    normalize_kwargs
```
Comment (laserkelvin) [inline docs/source/experiment-interface.rst:316]: Add a concrete pair of examples for the task args:

```yaml
normalize_kwargs:
    energy_mean: 0.0
    energy_std: 1.0
task_loss_scaling:
    energy: 1.0
```

**Review by laserkelvin — CHANGES_REQUESTED** [review]

File: docs/source/experiment-interface.rst (hunk 4 above)
Comment (melo-gonzo) [reply docs/source/experiment-interface.rst]: added defaults in [8b6b23f](https://github.com/IntelLabs/matsciml/pull/295/commits/8b6b23f11e0c205462d31cfbf6a1f08347191b11)

File: docs/source/experiment-interface.rst (hunk 3 above)
Comment (melo-gonzo) [reply docs/source/experiment-interface.rst]: added defaults in [8b6b23f](https://github.com/IntelLabs/matsciml/pull/295/commits/8b6b23f11e0c205462d31cfbf6a1f08347191b11)

File: docs/source/experiment-interface.rst (hunk 2 above)
Comment (melo-gonzo) [reply docs/source/experiment-interface.rst]: fixed in [8b6b23f](https://github.com/IntelLabs/matsciml/pull/295/commits/8b6b23f11e0c205462d31cfbf6a1f08347191b11)

File: docs/source/experiment-interface.rst (hunk 1 above)
Comment (melo-gonzo) [reply docs/source/experiment-interface.rst]: fixed in [8b6b23f](https://github.com/IntelLabs/matsciml/pull/295/commits/8b6b23f11e0c205462d31cfbf6a1f08347191b11)

**Review by laserkelvin — APPROVED** [review]
LGTM, thanks for doing this!

**Pull request merged**